src/injectscripts/jobs/situational_awareness.js

```diff
@@ -1,7 +1,541 @@
+    new Helicopter('7C617A', 'VH-TJG', 'RSCU203', 'AW-139'),
+    new Helicopter('7C617B', 'VH-TJH', 'RSCU204', 'AW-139'),
+    new Helicopter('7C617C', 'VH-TJI', 'RSCU206', 'AW-139'),
+    new Helicopter('7C617D', 'VH-TJJ', 'RSCU207', 'AW-139'),
+    new Helicopter('7C617E', 'VH-TJK', 'RSCU208', 'AW-139'),
+    new Helicopter('7C6182', 'VH-TJO', 'RSCU209', 'AW-139'),
+
+    // Careflight
+    new Helicopter('7C0635', 'VH-BIF', 'CFH4', 'BK117'),
+
+    // Westpac
+    new Helicopter('7C5CC0', 'VH-SLU', 'LIFE21', 'BK117'),
+    new Helicopter('7C81CC', 'VH-ZXA', 'WP1', 'AW-139'),
+    new Helicopter('7C81CD', 'VH-ZXB', 'WP2', 'AW-139'),
+    new Helicopter('7C81CE', 'VH-ZXC', 'WP3', 'AW-139'),
+    new Helicopter('7C81CF', 'VH-ZXD', 'WP4', 'AW-139'),
+
+    // PolAir
+    new Helicopter('7C4D03', 'VH-PHX', 'POLAIR1', 'EC-AS355'),
+    new Helicopter('7C4CF8', 'VH-PHM', 'POLAIR4', 'EC-135'),
+    new Helicopter('7C4D05', 'VH-PHZ', 'POLAIR5', 'Bell 412EPI'),
+
+    // Royal Australian Navy / CHC
+    new Helicopter('7C37B8', 'VH-LAI', 'CHOP22', 'Sikorsky S-76A'),
+    new Helicopter('7C44C8', 'VH-NVE', 'CHOP26', 'AW-139'),
+
+    // Some QLD based helicopters which may cross south
+    // QLD Westpac
+    new Helicopter('7C44CA', 'VH-NVG', 'LIFE45', 'EC-135'),
+
+    // QLD RAAF Rescue helicopter
+    new Helicopter('7C37B7', 'VH-LAH', 'CHOP41', 'Sikorsky S-76A'),
+
+    // RACQ Lifeflight
+    new Helicopter('7C759B', 'VH-XIL', 'RSCU511', 'AW139'),
+    new Helicopter('7C7599', 'VH-XIJ', 'RSCU533', 'AW139'),
+    new Helicopter('7C74C6', 'VH-XCO', 'RSCU588', 'Bell 412'),
+
+    // Some VIC base helicopters which may cross north
+    // Victoria Helicopter Emergency Medical Service (HEMS)
+    new Helicopter('7C7CC6', 'VH-YXK', 'HEMS1', 'AW-139'),
+    new Helicopter('7C7CC3', 'VH-YXH', 'HEMS2', 'AW-139'),
+    new Helicopter('7C7CC5', 'VH-YXJ', 'HEMS4', 'AW-139'),
+    new Helicopter('7C7CC1', 'VH-YXF', 'HEMS5', 'AW-139'),
+
+    // RFS fixed wing aircraft
+    new Helicopter('A4C031', 'N405LC', 'Thor', 'C130'),
+    new Helicopter('ACC37A', 'N512AX', '', 'DC-10'), // Bomber 910?
+    // Whoa, there appear to be a lot of these...
+];
+
+// Some extra data points for dev-time
+if (developmentMode) {
+    aircraft.push(
+        // ASNSW fixed wing
+        new Helicopter('7C41DE', 'VH-NAO', 'AM262', 'Super King 350C', false),
+        new Helicopter('7C41D9', 'VH-NAJ', 'AM292', 'Super King 350C', false), // Also seen as AM271
+        new Helicopter('7C01C2', 'VH-AMS', 'AM203', 'Super King B200C', false),
+        new Helicopter('7C01C1', 'VH-AMR', 'AM207', 'Super King B200C', false),
+        new Helicopter('7C01C0', 'VH-AMQ', 'AM297', 'Super King B200C', false),
+
+        // Royal Flying Doctor's Service
+        new Helicopter('7C3FE2', 'VH-MWK', 'FD286', 'Super King B200C', false)
+    );
+}
+
+// Load all the arcgis classes
+const GraphicsLayer = eval('require("esri/layers/GraphicsLayer");');
```

Comment:
I think I know why you're doing this. Please explain in a comment so future readers don't have to guess.